base/esg_postgres.py

```diff
@@ -148,17 +149,14 @@ def setup_db_schemas(publisher_password=None):
         esg_functions.set_publisher_password(publisher_password)
         db_user_password = esg_functions.get_publisher_password()
 
-    create_pg_super_user(cur, db_user_password)
 
     # create 'esgcet' user
-    create_pg_publisher_user(cur, db_user_password)
 
     # create CoG and publisher databases
```

Comment:
Probably can go ahead and delete these commented out lines if the testing was sufficient

Reply:
Will do. I commented out the call to `setup_db_schemas` in `setup_postgres`. We can talk about it at the meeting, but do you think `setup_db_schems` can be removed, as well as any functions it calls, as long as they are not being used elsewhere?

Reply:
Since you have tested, I think it is fine to be removed.  ```setup_db_schemas``` was not part of the classic installer.  It was a result of trying to model after the esgf-docker approach. Once I got it to work, I just never changed it.